src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 4;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        Color color = new ColorSupplier().getRandomColor();
+        int index = new Random().nextInt(FIGURE_COUNT);
+        Type figure = Type.values()[index];
+        if (figure.equals(Type.CIRCLE)) {
```

Comment:
we don't need to create a enum of figures, we can use index and switch-case to generate random figure.
Try to refactore this code